wifiphisher/common/interfaces.py

```diff
@@ -1,163 +1,147 @@
-#pylint: skip-file
 """
-This module was made to handle all the interface related operations for
-Wifiphisher.py
+This module was made to handle all the interface related operations of
+the program
 """
 
 import pyric
 import pyric.pyw as pyw
 
 
-class NotEnoughInterfacesFoundError(Exception):
-    """
-    Exception class to raise in case of a finding less than enough interfaces.
-    """
```

Comment:
Checking for the number of interfaces in not needed as I demonstrate with this example:

Cards:
- AP

This will throw an error stating that an interface with monitor mode is required but no found.

Cards:
- AP
- MON

There is no problem and would continue as normal.

Cards:
- MON


This will throw an error stating that an interface with AP mode is required but no found.